pypeit/core/wavecal/templates.py

```diff
@@ -434,7 +438,20 @@ def main(flg):
         wfile = os.path.join(template_path, 'Keck_LRIS', 'R600_5000', 'MasterWaveCalib_B_1_01.json')
         build_template(wfile, slits, lcut, binspec, outroot, lowredux=False)
 
-    if flg & (2**13):  # Magellan/MagE
+    if flg & (2**27):  # R600/7500
```

Comment:
See #850 , not necessarily for this PR.

Reply:
Good suggstion. And I will save it for a later day.